src/importexport/musicxml/tests/data/testNonStandardKeySig3.xml

```diff
@@ -22,6 +22,7 @@
       <part-abbreviation>Pno.</part-abbreviation>
       <score-instrument id="P1-I1">
         <instrument-name>Piano</instrument-name>
+        <instrument-sound>keyboard.piano</instrument-sound>
```

Comment:
and this a follow up, also not needed

Reply:
Seems I was wrong on that, sorry about this.

Reply:
Well, the test are green again …